charts/oes/charts/spinnaker/templates/configmap/halyard-init-script.yaml

```diff
@@ -44,16 +44,32 @@ data:
     # additionalInitScript
     {{ tpl .Values.halyard.additionalInitScript $ | indent 4 }}
     {{- end }}
-    {{- else }}
+    
+    {{- else if eq .Values.gitopsHalyard.repoType "git" }}
     #!/bin/bash -x
+    echo "+++++++++++running git commands+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++"
     git clone $GIT_CLONE_PARAM /tmp/spinnaker/.hal
     GIT_USER=`echo $GIT_USER | sed 's/ *$//g'`
     GIT_TOKEN=`echo $GIT_TOKEN | sed 's/ *$//g'`
-    DYNAMIC_ACCOUNTS_REPO=`echo $DYNAMIC_ACCOUNTS_REPO | sed 's/ *$//g'`
-    sed -i  s/SPINNAKER_NAMESPACE/${SPINNAKER_NAMESPACE}/ /tmp/spinnaker/.hal/config
+    {{- end }}
+    
+    {{- else }}
```

Comment:
These s3 instructions will be executed even if repo type is not s3, so it should be handled properly. Let's say repoType is not s3, not git, but "vault" even then this conditon will be hit and executed. So, handle it something like below

{{- else if eq .Values.gitopsHalyard.repoType "s3" }}